samples/client-sdk/knocking/index.js

```diff
@@ -125,6 +173,13 @@ const handleJoinedMeeting = (e) => {
   addParticipantVideo(participant);
 };
 
+const handleLeftMeeting = (e) => {
+  logEvent(e);
+  showForms();
+  hideVideos();
```

Comment:
this prevents the "rejected from call" message from being shown to the guest after getting denied.